smart-gate/packages/client/.env

```diff
@@ -1 +1,30 @@
-VITE_CHAIN_ID=31337
\ No newline at end of file
+# .envsample
+# ----------------------------------------
+# This file serves as a template for environment variables.
+# DO NOT commit changes made to this file.
+#
+# To use local environment variables:
+# 1. Duplicate this file.
+# 2. Rename the duplicate to .env.
+# 3. Update the .env file with your local environment settings.
+#
+# Example:
+# cp .envsample .env
+#
+# Add the .env file to your .gitignore to prevent accidental commits.
+
+VITE_CHAIN_ID=17069
+
+VITE_SMARTASSEMBLY_ID= #SSU ID
+
+# NEBULA
+VITE_GATEWAY_HTTP="https://blockchain-gateway-stillness.live.tech.evefrontier.com"
+VITE_GATEWAY_WS="wss://blockchain-gateway-stillness.live.tech.evefrontier.com"
+
+# STILLNESS
+# VITE_GATEWAY_HTTP="https://blockchain-gateway-stillness.live.tech.evefrontier.com"
+# VITE_GATEWAY_WS="wss://blockchain-gateway-stillness.live.tech.evefrontier.com"
+
+# # LOCAL
+# VITE_GATEWAY_HTTP="http://localhost:8080"
+# VITE_GATEWAY_WS="ws://localhost:8080"
```

Comment:
This is the .env file which should be in .gitignore. It's still picking up changes because of git caching. Run `git rm --cached .env` at the next commit